nullaway/src/main/java/com/uber/nullaway/handlers/CompositeHandler.java

```diff
@@ -252,13 +254,37 @@ public Optional<ErrorMessage> onExpressionDereference(
     return Optional.empty();
   }
 
+  /**
+   * An AccessPath predicate that always returns false. Used for optimizing
+   * getAccessPathPredicateForNestedMethod.
+   */
+  static final Predicate<AccessPath> FALSE_AP_PREDICATE = ap -> false;
+
+  /**
+   * An AccessPath predicate that always returns true. Used for optimizing
+   * getAccessPathPredicateForNestedMethod.
+   */
+  static final Predicate<AccessPath> TRUE_AP_PREDICATE = ap -> true;
```

Comment:
Why part of `CompositeHandler` vs `Handler`? Also, maybe we want `Handler.AccessPathPredicates.[TRUE|FALSE]_AP_PREDICATE`? (We can always static import to avoid repetition)

Reply:
See my comment above, I'm happy with making this change if you think it's better